### PR TITLE
Fixed javascript error when saving a recurrent provider event using js validation

### PR DESCRIPTION
**Pull request by sharonco-matrix**
There was a js error that prevents the recurrent event for provider to be saved when the javascript validation mechanism is on

### CODE REVIEW

**Comment by bradymiller** [comment]
Quick question. Isn't the javascript validation mechanism always on?

File: interface/main/calendar/add_edit_event.php [hunk 1]
```diff
@@ -119,7 +119,7 @@
     $collectthis = collectValidationPageRules("/interface/main/calendar/add_edit_event.php?group=true");
 } elseif ($_GET['prov']) {
     //providers tab
-    $collectthis = '';
+    $collectthis = collectValidationPageRules("/interface/main/calendar/add_edit_event.php?prov=true");;
```
Comment (bradymiller) [inline interface/main/calendar/add_edit_event.php]: extra semicolon at end of line

**Comment by sharonco-matrix** [comment]
Its controlled by the global. So its not always on. (we are talking about the new validation)

Fixed!

**Comment by bradymiller** [comment]
To clarify, though, the new validation only turns on/off this in the layouts module. As I recall, this is always turned on in the other forms (like this one) that are using it. correct?

**Comment by sharonco-matrix** [comment]
yes you are right :)

**Comment by bradymiller** [comment]
Code looks good and will bring into codebase. Thanks for the bug fix! -brady

**Pull request merged**